packages/devtools_app/lib/src/memory/memory_graph_model.dart

```diff
@@ -535,3 +538,138 @@ class HeapGraphClassLive extends HeapGraphClass {
   @override
   String toString() => name;
 }
+
+class InstructionsSize {
```

Comment:
nit: I'm not sure these classes should be in memory_graph_model (it was a big file).  memory_graph_model is used by table view and analysis and treemap.  These classes are only used by treemap?  

Maybe a treemap_model.dart to hold all of these classes.  These are very specific to treemap - right?

Reply:
For now, I've moved it to memory_treemap.dart, but I think the file structure can be evaluated again after hooking up search/selection/filter after this CL